src/processout/card.ts

```diff
@@ -661,8 +661,8 @@ module ProcessOut {
+                "bancontact": ["6703", "487104", "670305", "670330", "670319", "670300", "670342", "487109", "670397", "670375", "670310"]
```

Comment:
Adding `6703` and `670305` (and all the others) is redundant. As shared enclosed with a Bancontact BIN range analysis, we may be able to clean the code by just having the Maestro range (`6703`) and Visa (`487104`)
[bancontact.csv](https://github.com/processout/processout.js/files/11438589/bancontact.csv)